jobtracker/settings.py

```diff
@@ -112,6 +113,26 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.2/howto/static-files/
 
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, 'static'),
+]
+
+AWS_ACCESS_KEY_ID = 'AKIATKSN7JP4747J5FTU'
+AWS_SECRET_ACCESS_KEY = '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'
+AWS_STORAGE_BUCKET_NAME = 'jobtracker-static'
+AWS_S3_CUSTOM_DOMAIN = '%s.s3.amazonaws.com' % AWS_STORAGE_BUCKET_NAME
+
+AWS_S3_OBJECT_PARAMETERS = {
+    'CacheControl': 'max-age=86400',
+}
+
+AWS_LOCATION = 'static'
+STATICFILES_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+STATIC_URL = "https://%s/%s/" % (AWS_S3_CUSTOM_DOMAIN, AWS_LOCATION)
+
+DEFAULT_FILE_STORAGE = 'jobtracker.storage_backends.MediaStorage'
+
+
 STATIC_URL = '/static/'
```

Comment:
@jaywangjhw  It looks like STATIC_URL is getting defined twice. Is this going to interfere with our static css and javascript files? 
I know we had that issue with the static files when deploying before @luke-nam  so I'm tagging you here for input about that (since I'm not sure how you solved it)

Reply:
When I tested it, it didnt interfere with static css and javacript.